modules/commandModule/tests/testLogs/12.27.34_2021_04_04

```diff
@@ -1,80 +0,0 @@
-test_value_error_if_get_airspeed_equals_none (commandModule.tests.testCases.testReadNullFromPogi.TestReadingNullFromPOGIFiles) ... ok
```

Comment:
Okay unfortunately I am going to have to ask for some changes here, gitignore the testLogs folder, we don't want this committed to the codebase.